typescript/nft-multiple-currencies/nft.ts

```diff
@@ -0,0 +1,21 @@
+import { NATIVE_TOKEN_ADDRESS } from "@thirdweb-dev/sdk";
+
+export const nft = {
+  name: "My cool NFT",
+  description: "This is the description for my cool NFT",
+  image:
+    "https://images.unsplash.com/photo-1496181133206-80ce9b88a853?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1171&q=80",
```

Comment:
```suggestion
    "https://images.unsplash.com/photo-1496181133206-80ce9b88a853?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1171&q=80",
```

can we have a decentralized image plz

probably an ipfs hash and then use a gateway in the frontend, I don't like showing people how to upload centralized NFTs

Reply:
Oh okay, will use ipfs